File: OHelloDemo/OHello/OHello/ViewController.m
//
//  ViewController.m
//  OHello
//
//  Created by screson on 2017/4/10.
//  Copyright © 2017年 screson. All rights reserved.
//

#import "ViewController.h"

NSString * textHello = @"Hello,World";

@interface ViewController ()
{
    NSString * userName;
}

@end

@implementation ViewController

+ (void)HelloWorld{
    NSLog(@"%@",textHello);
}

- (void)viewDidLoad {
    [super viewDidLoad];
    userName = @"JuanFelix";
    NSLog(@"%@",userName);
    [self helloWorld];
    [ViewController HelloWorld];
    [self popSort];
}

- (void)helloWorld{
    NSLog(@"%@",textHello);
}

- (void)popSort{
    int a[5] = {1,43,21,3,4};
    for (int i = 0; i < 5; i++) {
        for (int j = i; j < 5; j++) {
            if (a[j] < a[i]) {
                int temp = a[j];
                a[j] = a[i];
                a[i] = temp;
            }
        }
    }
    for (int n = 0; n < 5; n ++) {
        NSLog(@"%d",a[n]);
    }
}

@end

File: SHelloDemo/SHello/SHello/ViewController.swift
//
//  ViewController.swift
//  SHello
//
//  Created by screson on 2017/4/10.
//  Copyright © 2017年 screson. All rights reserved.
//

import UIKit

let textHello = "Hello,World"

class ViewController: UIViewController {
    var userName:String!
    
    class func HelloWorld(){
        print(textHello)
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        userName = "JuanFelix"
        print(userName)
        self.helloWorld()
        ViewController.HelloWorld()
        self.popSort()
    }
    
    func helloWorld() {
        print(textHello)
    }
    
    func popSort() {
        var a = [1,43,21,3,4]
        for i in 0..<5 {
            for j in i..<5{
                if a[j] < a[i]{
                    let temp = a[j]
                    a[j] = a[i]
                    a[i] = temp
                }
            }
        }
        for i in 0..<5 {
            print(a[i])
        }
    }
}

